osbuild/pipeline.py

```diff
@@ -369,10 +383,11 @@ class Manifest:
     def __init__(self):
         self.pipelines = collections.OrderedDict()
         self.sources: List[Source] = []
+        self.runners: Dict[str, Runner] = {}
```

Comment:
Where is that field used?

Reply:
Ups, left over from the start of my refactoring to work around the mypy issue. Good catch, thanks!